test/acl_auto_configure_test.cpp

```diff
@@ -1490,14 +1490,16 @@ TEST(auto_configure, cra_ring_root_exist) {
 
 TEST(auto_configure, hostpipe_mappings) {
   const std::string config_str{
-      "23 66 " RANDOM_HASH
+      "23 71 " RANDOM_HASH
       " pac_a10 0 1 13 DDR 2 2 24 1 2 0 4294967296 4294967296 8589934592 0 - 0 "
-      "0 0 0 0 0 1 5 8 pipe_logical_name1 pipe_physical_name1 1 12345 0 1 4 10 "
-      "pipe_logical_name2 pipe_physical_name2 0 12323 1 0 8 20 "
+      "0 0 0 0 0 1 5 9 pipe_logical_name1 pipe_physical_name1 1 12345 0 1 4 10 "
```

Comment:
Nit: would you mind format the string so that the character on each line roughly aligns? It seems a little hard to read at this point

Reply:
I will fix it. It was alright but too long, then clang-formatting over and over makes harder to read. I will make it nice to play with clang-format.